.pipelines/templates/e2e-windows-template.yaml

```diff
@@ -111,4 +111,4 @@ jobs:
         MAPPED_ACCOUNT_KEY: $(STORAGE_ACCOUNT_KEY)
     - publish: $(System.DefaultWorkingDirectory)/${{ parameters.windowsImage }}/$(SCENARIO_NAME)-logs
       artifact: ${{ parameters.windowsImage }}-$(SCENARIO_NAME)-logs
-      condition: always()
\ No newline at end of file
+      condition: succeeded()
```

Comment:
Does it mean that we will not upload logs if it failed? I thought that the logs would help us for troubleshooting.

Reply:
Thanks for the comment. I will keep the condition of cse logs as always() for debug.